src/gam.py

```diff
@@ -3859,11 +3816,10 @@ def getPhoto(users):
       photo_data = photo['photoData']
       if showPhotoData:
         print(photo_data)
-      photo_data = base64.urlsafe_b64decode(photo_data)
     except KeyError:
       print(' no photo for %s' % user)
       continue
-    writeFile(filename, photo_data, mode='wb', continueOnError=True)
+    writeFile(filename, base64.urlsafe_b64decode(photo_data), mode='wb', continueOnError=True)
```

Comment:
let's keep this as two lines to make it easier to read and under 80 char length.